projects/fx-Protocol/index.js

```diff
@@ -32,7 +34,19 @@ async function getGaugeTvl(api) {
   })
 }
 
+async function getFxProtocolPoolManagerStrategyTvl(api) {
+  const bals = await api.call({ abi: 'uint256:totalSupply', target: FxProtocol_PoolManager_AaveV3Strategy })
+  api.add("0xc035a7cf15375ce2706766804551791ad035e0c2", bals)
+}
+
+async function getFxProtocolStabilityPoolStrategyTvl(api) {
+  const bals = await api.call({ abi: 'uint256:totalSupply', target: FxProtocol_PoolManager_AaveV3Strategy_USDC })
```

Comment:
can we use erc20:balanceOf for these instead?

Reply:
OK